Cargo.toml

```diff
@@ -55,4 +55,4 @@ serde_json = "1"
 base64 = "0.22.1"
 chrono = { version = "0.4.38", default-features = false }
 itertools = "0.13"
-pyo3 = { version = "0.21.2", features = ["experimental-async"] }
+pyo3 = { version = "0.22.2", features = ["experimental-async"] }
```

Comment:
did this dep need to be updated? I think datafusion-python currently requires pyo3 0.21